Kickstarter-iOS/Features/Comments/Controllers/CommentsViewControllerTests.swift

```diff
@@ -174,35 +174,6 @@ internal final class CommentsViewControllerTests: TestCase {
     }
   }
 
-  func testView_CurrentUser_LoggedIn_IsBacking() {
```

Comment:
Always always always remember to remove screenshots if a test is removed. :)

Reply:
sorry. those are gone now.